### ISSUE TITLE
Switching off LED

### ISSUE THREAD

**Issue by seppelicous** [cp2004/OctoPrint-WLED#28]
How do I switch off the LEDs, eg end of print? Do I have to set brightness to 0 or what else?

**Comment by cp2004** [comment]
Currently, setting the print finished event to 0 brightness or solid colour black would achieve that. The way I managed this with WS281x LED Status was using @ commands so you could add them to gcode scripts etc. to control the lights on/off.

**Comment by Nawor3565** [comment]
Hi! I'm in the same boat, I would like to have the plugin make the LEDs flash for a couple seconds to alert me when a print is done, but not keep blinking perpetually. The ability to use g-code commands would be perfect for doing that (or just a checkbox option that turns off the LEDs a few seconds after the effect is triggered). Thanks!

**Comment by cp2004** [comment]
In version 0.2.0 there will be two @ commands available:

> Host @ commands can be used in GCODE scripts, custom controls or sent directly from the OctoPrint terminal to control OctoPrint. The WLED plugin reacts to commands that start with `@WLED`:
> * `@WLED ON` turns all the LEDs on
> * `@WLED OFF` turns all the LEDs off
> These are not case sensitive.

**Comment by NovaViper** [comment]
I want to try this out for OctoDash!

**Comment by cp2004** [comment]
@NovaViper I've tagged a development release as [0.2.0dev1](https://github.com/cp2004/OctoPrint-WLED/releases/tag/0.2.0dev1)

Install using the URL:
```
https://github.com/cp2004/OctoPrint-WLED/releases/download/0.2.0dev1/release.zip
```

**Comment by cp2004** [comment]
There's a release candidate available for 0.2.0rc1 if you want to give it a go. Feedback ticket #54.

**Comment by cp2004** [comment]
0.2.0 has been released
